### PR TITLE
SEARCH-825 (Create a helm chart for Alfresco Zeppelin)

### PR DESCRIPTION
**Pull request by tunaaksoy**

### CODE REVIEW

**Comment by tunaaksoy** [comment]
requirements.yaml points to incubator for the time being. It'll be fixed once we have all the docker images and helm charts released.

File: helm/alfresco-content-services/requirements.yaml [hunk 1]
```diff
@@ -10,8 +10,8 @@ dependencies:
   repository: https://kubernetes-charts.storage.googleapis.com/
   condition: postgresql.enabled
 - name: alfresco-search
-  version: 0.0.3
-  repository: http://kubernetes-charts.alfresco.com/stable
+  version: 0.0.4
+  repository: http://kubernetes-charts.alfresco.com/incubator
```
Comment (derekhulley) [inline helm/alfresco-content-services/requirements.yaml]: We will need to keep a close eye on how we use incubator vs having stable EA candidates.  This is something we will have to figure out over time, so just worth keeping an eye on.

**Comment by tunaaksoy** [comment]
@amohammedalfresco as discussed last week, I moved the network policy for Zeppelin to ACS helm chart. I also needed to do some changes in the network policies for Repo and Search. Can you review the changes please?

@derekhulley I need these changes to be merged and published. First I have to release the helm charts for Zeppelin and Search though. After that I'll update requirements.yaml. Can we catch up on Monday and discuss this please?

**Comment by derekhulley** [comment]
@tunaaksoy , you have 3 LGTMs.  In our workflow, that means you can perform the merge.

**Comment by tunaaksoy** [comment]
Thank you all for the code reviews. I've pushed all the code changes we need for Zeppelin deployment to this feature branch and the bamboo build for it is green as well.

I just need to talk to someone from the Repo team if master is the right branch to merge to as we need a new ACS deployment helm chart release with these changes in. I suspect I should create a branch from 1.0.1 and push the changes there, then create a release (v 1.0.2) and after that merge the changes to master. Can someone confirm if these are the correct steps to do a release please?

**Comment by killerboot** [comment]
Pleas emerge to master and then cherry-pick to support/HF/1.0. I will then release 1.0.2 chart.

**Comment by tunaaksoy** [comment]
@killerboot I've merged the changes to master and cherry picked them to a branch of support/HF/1.0. It requires code review there as well (see #42). The bamboo build for the hotfix branch is running at the moment. In 10-15 mins it should be ready to be merged to support/HF/1.0.

**Pull request merged**